test/compress/parameters.js

```diff
@@ -1,4 +1,7 @@
 arrow_functions: {
+    options = {
+        arrows: true,
```

Comment:
I've shoved `OPT(AST_Arrow)` under the same flag - you cannot switch these rules off before.